corehq/apps/userreports/reports/data_source.py

```diff
@@ -102,13 +107,22 @@ def get_data(self, slugs=None):
             ProgrammingError,
         ) as e:
             raise UserReportsError(e.message)
-        # arbitrarily sort by the first column in memory
-        # todo: should get pushed to the database but not currently supported in sqlagg
+        # TODO: Should sort in the database instead of memory, but not currently supported by sqlagg.
```

Comment:
Originally I was hoping to move the sorting into the DB layer, but decided to take the easy route here because there are a few challenges to implementing DB level sorting in sqlagg (sqlagg queries can span multiple tables, return format is currently a dictionary).